main.go

```diff
@@ -102,6 +110,43 @@ func handleUserRequest(responseWriter http.ResponseWriter, request *http.Request
 	}
 }
 
+func handleSessionRequest(
+	responseWriter http.ResponseWriter,
+	request *http.Request) {
+
+	type LoginForm struct {
+		EmailAddress string `json:"emailAddress"`
+		Password     string `json:"password"`
+	}
+
+	switch request.Method {
+	case http.MethodPost:
+		var loginForm LoginForm
+		body := getRequestBody(request)
+
+		if err := json.Unmarshal(body, &loginForm); err != nil {
+			panic(err)
+		}
+
+		validated, err := databaseutil.ValidateUser(
```

Comment:
booleans should more of a statement: `isValidated` or maybe `userIsValidated` or `isValidatedUser`

Reply:
bump